src/settings.cpp

```diff
@@ -2016,6 +2016,22 @@ void IConsoleListSettings(const char *prefilter)
 	IConsolePrintF(CC_WARNING, "Use 'setting' command to change a value");
 }
 
+/**
+ * Get the SaveLoad description for the SettingTable.
+ * @param settings SettingDesc struct containing all information.
+ * @return Vector with SaveLoad entries for the SettingTable.
+ */
+static std::vector<SaveLoad> GetSettingsDesc(const SettingTable &settings)
```

Comment:
Does this function need to filter for SLF_NOT_IN_SAVE, or is that done somewhere else?

Reply:
Clearly this is done somewhere else :D I basically only moved code to make it more readable.

Within `saveload.cpp` these flags are handled, similar to savegame version ranges etc. Of course we could also do it in this function, but that only bloats the function.

`SlIsObjectValidInSavegame` is the word to look for in `saveload.cpp` for the details.

Reply:
Ok, so SlObject deals with it. But SlCalcTableHeader only copies "from" and "to", and not "conv", does it?